cmd/minishift/cmd/start.go

```diff
@@ -156,14 +164,6 @@ func runStart(cmd *cobra.Command, args []string) {
 		atexit.Exit(1)
 	}
 
-	// Set Proxy to as Shell Env
```

Comment:
If we remove it, we also need to adjust the docs which say that the proxy settings are set on a system level when logging in.